### DISCUSSION TITLE
RVC usage in this repository.

### DISCUSSION THREAD

**Discussion by fnt-faizan** [serp-ai/bark-with-voice-clone#102] [Q&A]
Has anyone used the RVC feature yet in this repo because  the code for using RVC is incomplete?

**Comment by devinschumacher** [comment]
We just launched our follow up to BARK -- an AI Voice Cloner Desktop App (runs locally, offline, super realistic voice cloning, data privacy, no coding needed)

👉Check it out: https://www.youtube.com/watch?v=84JVGtkapik

> Has anyone used the RVC feature yet in this repo because the code for using RVC is incomplete?
